src/dmd/globals.d

```diff
@@ -261,6 +261,10 @@ extern (C++) struct Param
     const(char)[] resfile;
     const(char)[] exefile;
     const(char)[] mapfile;
+
+    bool timeTrace;
+    uint timeTraceGranularity = 500;
```

Comment:
a comment as to the units  of `timeTraceGranularity` would be good here. Is this microseconds or milliseconds or something else?

Reply:
noted. I need to update the manual too presumably. it's 4am now so that's something for tomorrow.

Reply:
I recommend to add the unit as postfix to the name of the variable.